relnotes/pkg-iteration.feature.md

```diff
@@ -0,0 +1,21 @@
+## The package iteration can now be specified via `PKGITERATION`
+
+Package versions consist of two components: the _upstream version_, and
+the _package iteration_ (corresponding to the `debian_revision` where
+deb packages are concerned).
+
+Previously the package iteration was hardcoded to match the distro code
+(e.g. `xenial`), disallowing typical patterns referencing the packager
+and changes to the packaging rather than the upstream project.  For
+example, Ubuntu packages typically use an `XubuntuY` format where `X`
+is the numerical revision of the Debian package that has been forked
+(or `0` if there is no such package), and `Y` is the Ubuntu package
+revision (hence Ubuntu package versions like `2.7.0-2ubuntu3.1`, where
+`2.7.0` is the upstream version and `2ubuntu3.1` means Ubuntu revision
+3.1 of Debian package revision 2).
```

Comment:
Not a huge deal, and I'm fine if you want to leave this, but I think the whole thing about how ubuntu is using encoding the `debian_revision` is irrelevant and distracting.